ResetPassword.js

```diff
@@ -73,11 +73,11 @@ module.exports = function (Model, options) {
 
   Model.on('resetPasswordRequest', function (info) {
     var settings = Model.app.settings;
-    var html = 'Click on <a href="' + settings.protocol + '://' + settings.host + ':' + settings.port + '/confirm-password-reset?access_token=' + info.accessToken.id + '">this</a> url to reset your password';
+    var html = 'Hello <br /><br /> There was recently a request to change the password on your account.<br /><br />If you requested this password change, please click the <a href="\' + settings.protocol + \'://\' + settings.host + \':\' + settings.port + \'/confirm-password-reset?access_token=\' + info.accessToken.id + \'">link</a> to set a new password url to reset your password.<br /><br />If you did not make this request or need assistance pls write to us at support@bnext.in or call our customer support no. +91 992 099 8066<br /><br />Thank you.<br />The Bnext Team';
```

Comment:
@sachinbreakingthings It's a generic repo, not specific to bnext. Either keep the content generic, or better even, accept the HTML as an input.